src/Sia.Data.Incident/Filters/EventFilters.cs

```diff
@@ -28,6 +30,11 @@ public override IQueryable<Event> Filter(IQueryable<Event> source)
             if (EventTypes != null && EventTypes.Length > 0) working = working.Where(ev => EventTypes.Contains(ev.EventTypeId));
             if (Occurred.HasValue) working = working.Where(ev => ev.Occurred == Occurred);
             if (EventFired.HasValue) working = working.Where(ev => ev.EventFired == EventFired);
+            if (StartTime.HasValue && EndTime.HasValue)
```

Comment:
These checks should follow the same pattern as the others; if we have a start time, compare against start time, and if we have an end time, compare against end time. If a client wants to send a request for the last 10 minutes of events (up to the most recent events received), it doesn't make sense to force them to provide an end time. Similarly, if a client wants to send a request for every event related to a given incident that happened before the detection of an incident, it doesn't make sense to force them to provide a start time.